docs/_tutorials/messagevpn-with-queue-java.md

```diff
@@ -0,0 +1,246 @@
+
+```java
+switch (command)  {
+    case "createvpn":
+        app.createMessageVpn(messageVpnName);
+        app.updateDefaultClientProfileForPersistentMessaging(messageVpnName);
+        app.setupClientUsername(messageVpnName, vpnUserName, vpnUserPassword);
+        // Additionally create a queue
+        app.createQueue(messageVpnName, testQueueName);
+        break;
+    case "deletevpn":
+        app.deleteMessageVpn(messageVpnName);
+        break;
+```
+
+## Summary
+
+The full source code for this example is available in [GitHub]({{ site.repository-java }}){:target="_blank"}. If you combine the example source code shown above results in the following source:
+
+```
+ManageVPN.java
+```
+
+### Getting the Source
+
+Clone the GitHub repository containing the Solace samples.
+
+```
+git clone https://github.com/SolaceSamples/solace-samples-semp
+cd solace-samples-semp/Java
+```
+
+### Building
+
+Building the example is simple. You can simply build the project using Gradle.
+
+The build instructions in this tutorial assume you are using a Linux shell. If your environment differs, adapt the instructions.
+
+```
+./gradlew assemble
+```
+
+This builds the manageVPN tool with OS specific launch scripts. The files are staged in the `build/staged` directory.
+
+### Running the Sample
+
+The `manageVPN` code has the management user and password hardcoded as *admin:admin*. Change that in the code if required and re-build.
+
+From the `solace-samples-semp/Java` directory, replace `<host:port>` by the VMR host name or IP address and the management port (the default port for the VMR is 8080) and execute:
+
+```
+build/staged/bin/manageVPN createvpn <host:port> myNewVPN
+```
+
+This will create a new message-VPN called `myNewVPN` and also create a sample queue called `testQueue`. The credentials to connect to the new message-VPN are *default:password*, which can also be changed in the code if required.
+
+A message-VPN can also be deleted when no longer needed:
+
+```
+build/staged/bin/manageVPN deletevpn <host:port> myNewVPN
+```
+
+This will detect that the queue `testQueue` and possibly other queues are still there so it will delete these first and then it will delete `myNewVPN`.
+
+Now it is possible to easily create as many message-VPNs as required and share an installed VMR with other developers. To try a new message-VPN, the [Persistence with Queues]({{ site.solace-samples-jms-queues }}){:target="_top"} sample has been enabled to take the additional optional parameter of a message-VPN name and will use that for messaging.
+
+```
+cd <parent_dir_outside_of_solace-samples-semp>
+git clone https://github.com/SolaceSamples/solace-samples-jms
+cd solace-samples-jms
+./gradlew assemble
+./build/staged/bin/queueProducer <host> myNewVPN
+```
```

Comment:
Better to point off to the Persistent with queues sample rather than replicate parts of the tutorial here which may eventually fall out of date.